script.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-dupe-class-members */
+/* eslint-disable linebreak-style */
+
```

Comment:
- [ ] [OPTIONAL] Please make sure you fix your linter errors instead of disabling them. This would help you learn best practices and would help you get used to such errors 🙏.